### ISSUE TITLE
fdconfig.sys SHELL=4debug.com will try to load debug.com

### ISSUE THREAD

**Issue by ecm-pushbx** [FDOS/kernel#121]
With a file named `4debug.com` on the boot drive and `shell=4debug.com` in the fdconfig.sys file, the following interaction happens at boot time:

```
Bad or missing Command Interpreter: debug.com
Enter the full shell command line: 4debug.com
-
```

As seen, entering the full filename including the leading digit does allow to load the shell. But the kernel first tries to load the shell with the digit stripped. Using a name like `ldebug.com` without the leading digit works as expected.

If I recall, this was first reported on one of the mailing lists by a user trying to load 4DOS. I can't look that up right now however.

I suspect that the digit is read or discarded like a menu selector digit. I have not studied this in detail however. It doesn't explain why the `shell=` line does take effect even though it processes the `4` however.

**Comment by ecm-pushbx** [comment]
Workaround: Insert a backslash, which has no effect but will avoid this problem.

**Comment by ecm-pushbx** [comment]
The original report was actually on our forum, https://www.bttr-software.de/forum/forum_entry.php?id=21218

**Comment by PerditionC** [comment]
fixed by pr#129
